setup.py

```diff
@@ -76,7 +76,11 @@
     install_requires=[
         'google-cloud-storage>=1.35,<1.44',
         'matplotlib>=3.3,<3.6',
+        'tensorflow_datasets>=4.2,<4.5',
```

Comment:
I think this is covered by tensorflow_deps above? Installing with `pip install -e .[all]` should install all the dependencies (not just the core ones).

Reply:
I meant keras_deps

Reply:
This is a relic from when I was hacking around with docker containers